app/client/src/features/accounts/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable react/jsx-curly-newline */
```

Comment:
The order of the imports should be fixed here.  I need to add an eslint rule, but in general we go 

```js
import { Foo } from 'npm-package';
// new line
import { Bar }  from 'local-code-that-we-wrote';
```